spec/javascript/components/lease_ups/__snapshots__/LeaseUpsPage.test.js.snap

```diff
@@ -485,7 +485,7 @@ Array [
                   }
                 >
                   <div
-                    className="dropdown"
+                    className="dropdown "
```

Comment:
do we know why the classname has a space in it now?

Reply:
We are doing this `dropdown ${wrapperClasses ? wrapperClasses.join(' ') : ''}`` in here
https://github.com/Exygy/sf-dahlia-lap/pull/111/files#diff-97e2ac7b9ff31ba87ab51ccba57ee7fbR89

We need the extra space in there to render `wrapperClasses` if present. If it not present it will still render the space. It can be fixed using `classNames` lib.  However, it will feel a bit inconsistent because we are using the `join` approach in multiple places. 
@callaghanc any thoughts?

Reply:
@fddayan We can use `classNames` in this case - would you mind updating to use that?